ui/scss/forms-engine.scss

```diff
@@ -116,6 +116,10 @@ div.xdsoft_datetimepicker {
   z-index: 1040 !important;
 }
 
+.tox-fullscreen .tox.tox-tinymce-aux, .tox-fullscreen~.tox.tox-tinymce-aux {
+  z-index: 1201 !important;
+}
```

Comment:
What's this for? Why is this needed?

Reply:
While testing I found an issue in fullscreen mode where the menu bar items (when clicking on an entry) were not showing

Reply:
**Why**? Instead of adding importing to this one, should we be getting rid of other styles elsewhere?